src/components/StatsScoringDialog.vue

```diff
@@ -1,90 +1,90 @@
 <template>
-  <v-dialog v-model="show">
+  <base-dialog
+    v-model="show"
+    title="Ranked Scoring"
+    :id="`rankedScoring`"
+    max-width="1250"
+  >
     <!-- Activator -->
     <template #activator="{ props }">
       <slot name="button">
         <v-btn
           v-bind="props"
-          color="primary"
+          color="surface-2"
```

Comment:
This is the activator button that's used in the StatsView, right? I don't think we need to update this color yet until we rework the stats view itself
```suggestion
          color="primary"
```